pycontrails/core/fleet.py

```diff
@@ -341,19 +341,6 @@ def segment_angle(self) -> tuple[npt.NDArray[np.float_], npt.NDArray[np.float_]]
         cos_a[self.final_waypoints] = np.nan
         return sin_a, cos_a
 
-    @overrides
-    def fit_altitude(
```

Comment:
I think you should keep this, but change the method name to `clean_and_resample`. If we call this new method on on a `Fleet`, we want some sort of error.